I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 33, in <module>
    if not povename:
NameError: name 'povename' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "lib/windows-i686/script.rpyc", line 29, in script
  File "C:\Users\Intel\Desktop\Emily Folder\renpy-6.99.8-sdk\renpy\ast.py", line 805, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Intel\Desktop\Emily Folder\renpy-6.99.8-sdk\renpy\python.py", line 1461, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 33, in <module>
    if not povename:
NameError: name 'povename' is not defined

Windows-7-6.1.7601-SP1
Ren'Py 6.99.8.959
Game Dev Intern 0.0
